html-css/week1/styles.css

```
@@ -0,0 +1,296 @@
+body {
+    font-family: Arial, sans-serif;
+    width: 100%;
+    margin: 0;
+    background: linear-gradient(hsl(240, 22%, 50%), hsl(186, 26%, 52%), hsl(222, 12%, 45%));
+}
+
+nav {
+    position: fixed;
+    display: flex;
+    align-items: center;
+    width: 100%;
+    height: 5rem;
+    background-color: hsl(240, 68%, 37%);
+}
+
+nav ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    gap: 10%;
+    width: 100%;
+    list-style-type: none;
+    margin-left: -4%;
+}
+
+nav ul li {
+    font-size: 24px;
+    color: #dfdfe2;
+}
+
+nav ul li:hover {
+    background-color: lightgreen;
+    color: blue;
+    cursor: pointer;
+}
+
+li> a {
+    color: inherit;
+    text-decoration: none;
+    font-weight: bold;
+}
+
+header {
+    padding-top: 110px;
+    width: 100%;
+    background: linear-gradient(hsl(240, 56%, 42%), hsl(186, 76%, 16%), hsl(223, 90%, 60%));
+}
+
+header h1, p {
+    color: rgb(136, 187, 136);
+    letter-spacing: 1.5px;
+    text-align: center;
+}
+
+header p {
+    font-style: italic;
+    padding-bottom: 20px;
+}
+
+#welcome-selection {
+    width: 100%;
+}
+
+.welcome-section {
+    display: flex;
+    flex-direction: column;
+    padding: 20px 7% 20px 7%;
```

There is a practice called short-hand, where you can write this as:
```suggestion
    padding: 20px 7%;
```

https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties